registry/storage/blobstore.go

```diff
@@ -141,6 +141,7 @@ func (bs *blobStore) resolve(ctx context.Context, path string) (string, error) {
 type blobStatter struct {
 	driver driver.StorageDriver
 	pm     *pathMapper
+	tomb   *tomb
```

Comment:
I don't think having the `tomb` at this level makes sense. Tombstones are repository bound, whereas this `blobStatter` operates on the global storage.